### ISSUE TITLE
Redesign and reposition of certain sub-headings in accessEd and duggaEd.

### ISSUE THREAD

**Issue by a18ismca** [HGustavs/LenaSYS#8131]
Based on issue #8109 where changes were made to resultEd, but not elsewhere.

**Description of #8109:**
In resultEd in LenaSYS, there is a heading called Result. It would be good to rename to heading to a more appropriate name, for example "Dugga Results for Students" or "Result Editor". Also, it would be good that the heading is replaced more to the left of the screen, above the LadExport table to increase readability.

**NOTE: In this case resultEd is replaced by the affected editors listed in the rubrik.**
Link to the based issue: https://github.com/HGustavs/LenaSYS/issues/8109

TL;DR: Move some headings to the left side of some pages instead of having them centered.

**Comment by b18alegu** [comment]
Note that the file editor is not our groups responsibility, please contact group 3 before making any changes. I suggest to make a separate pull request for their changes and let them merge that into their weekly branch.

**Comment by b18alegu** [comment]
Closed after successful weekly-merge.
